javascript/node/selenium-webdriver/lib/until.js

```diff
@@ -120,6 +126,31 @@ exports.alertIsPresent = function alertIsPresent() {
   });
 };
 
+/**
+ * Creates a condition for checking that an element is present on the DOM of a page and visible.
+ * Visibility means that the element is not only displayed but also has a height and width that is
+ * greater than 0.
+ *
+ * @param {!By} locator used to find the element
+ * @return {!Condition<WebElement>} the WebElement once it is located and visible
+*/
+exports.visibilityOfElementLocated = function(locator) {
+  return new Condition('visibility of element located by ' + locator.toString(), function(driver) {
+      return driver.findElement(locator).then(element => {
```

Comment:
could you use async/await to avoid promise chaining?

Reply:
You mean only this statement or is it for the other changes too? The other functions use promise chaining too, even though they don't have 2 chains like this one.